src/custom-lists/components/overview/sidebar/Index.js

```diff
@@ -118,9 +122,12 @@ class ListContainer extends Component {
                     onEditButtonClick={this.props.handleEditBtnClick(i)}
                     onListItemClick={this.props.handleListItemClick(list, i)}
                     onAddPageToList={this.props.handleAddPageList(list, i)}
+                    onAddSubListToList={this.props.handleAddSubList(list, i)}
                     onCrossButtonClick={this.props.handleCrossBtnClick(list, i)}
                     resetUrlDragged={this.props.resetUrlDragged}
-                />
+                >
+                    {list.subList && this.renderAllLists(list.subList)}
```

Comment:
Going by how you're using `props.lists[].subList`, it seems like that `props.lists` prop is an array of list types recursively defined something like this

```ts
interface List {
  ...
  subList: List[]
}
```

Is that right? If so, that should ok, but I'd rename that property to the plural `subLists`

Reply:
Great, i'll rename it.